src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/EntityStatus.java

```diff
@@ -21,10 +21,15 @@
 		RecyclableArrayList<ClientBoundPacketData> packets = RecyclableArrayList.create();
 		if(allowedIds.contains(status)) {
 			NetworkEntity e = cache.getWatchedEntity(entityId);
-			//The only remap I know so far.
-			if((status == 31) && (e.getType() == NetworkEntityType.FISHING_FLOAT)) {
+			if ((status == 31) && (e.getType() == NetworkEntityType.FISHING_FLOAT)) {
 				status = 13;
 			}
+			if ((status == 10) && (e.getType() == NetworkEntityType.MINECART_TNT)) {
+				status = 31;
+			}
+			if ((status == 17) && (e.getType() == NetworkEntityType.FIREWORK)) {
```

Comment:
I guess we will need a remap table for this soon

Reply:
Yeah It's already a todo in there :S